src/index.ts

```diff
@@ -1,11 +1,110 @@
+//expressおまじない
 import express from 'express'
 
+import bodyParser from 'body-parser'
+
+// import fs from 'node:fs'
+import sqlite from 'sqlite3'
+
 const app: express.Express = express()
+const port = 3000
+app.use(bodyParser.json())
+
+// const initDataSql = fs.readFileSync('./sql/sampleData.sql').toString()
+// const dummyDataSqls = fs
+//   .readFileSync('./sql/sampleData.sql')
+//   .toString()
+//   .split(';')
+//   .slice(0, -1)
+
+const db = new sqlite.Database('./data/data.db', (err) => {
+  if (err) {
+    console.error(err)
+    throw Error('データベースが開けませんでした')
+  }
+  console.log('データベースが開かれました')
+
+  db.run(
+    'CREATE TABLE IF NOT EXISTS users (id INTEGER PRIMARY KEY AUTOINCREMENT, userID TEXT NOT NULL);',
+    (err) => {
+      if (err) {
+        console.error(err)
+        throw Error('テーブルの作成に失敗しました')
+      }
+      console.log('テーブルが作成されました')
+    },
+  )
+
+  db.run(
+    'CREATE TABLE IF NOT EXISTS close_table (userId TEXT PRIMARY KEY, close INTEGER NOT NULL);',
+  )
+})
+
+app.get('/user/:userID', (req: express.Request, res: express.Response) => {
```

Comment:
```suggestion
app.post('/user/:userID', (req: express.Request, res: express.Response) => {
```
INSERT するなら POST リクエストの方が良さそう

Reply:
これは @gentakahasi のコードなので、 @gentakahasi がプルリクを出したときに指摘してほしいやつ（もしかしたら本プルリクで直してるかもしれないし）

Reply:
それなら `main` への PR は適切ではないような

Reply:
（それはそう）DBまわりはタスクから分離するべきだったかな、と思う次第

少なくともこの部分についてのレビューは「このプルリクエスト内ではスコープ外」だと思っているつもり

これについてよい運用があれば教えてほしい

Reply:
Request 先をcreate元である `develop-gosan` に変更するとか？
この先どのように進めていくかにもよるだろうけど、許容してもよいのであれば `main` へmergeしちゃっても良いのかな。今すでに運用しているサービスっていうわけでもないし
（こういうのは、コードによっては変更忘れがあるとちょっと怖いけど）

それくらいしか思いつかないなぁ